README.md

```diff
@@ -20,97 +18,68 @@ In order to create unsigned certificates, you must first follow the instructions
-    # To use a sample unsigned certificate as follows:
-    cp /cert-issuer/examples/data-testnet/unsigned_certificates/3bc1a96a-3501-46ed-8f75-49612bbac257.json /cert-issuer/data/unsigned_certificates/ 
-    ```
-    
-3. Make sure you have enough ETH in your issuing address.
-
-    a. Go to https://myetherwallet.com and create a public and private address. Make sure to store the private address in a secure location.
-    b. Go to https://faucet.bloxberg.org and enter your created public address to receive bergs. 
-
-4. Further configure the settings for cert-issuer in conf.ini
-
-    ```
-    # public address
-    issuing_address = <issuing-address>
-
-    # issuer Public Key
-    verification_method = <verification_method>
-    
-    # Chain certificates will issue to
-    chain=ethereum_bloxberg
-    
-    # Text file containing your private key
-    usb_name = </Volumes/path-to-secure-location/>
-    key_file = <file-you-saved-private-key-to>
-    
-    #Path to certificate files, set to default but can be changed
-    unsigned_certificates_dir=./data/unsigned_certificates
-    blockchain_certificates_dir=./data/blockchain_certificates
-    
-    #use an ERC721 smart contract for issuance, leave this as default
-    issuing_method = smart_contract
-  
-    #node url to broadcast transactions
-    node_url = https://core.bloxberg.org
-    
-    #ens_name is set by the cert-deployer repository
-    ens_name = <mpdl.berg>
-    
-    #default path for revocations - currently revocations aren't supported.
-    revocation_list_file=./revocations.json
-
-    no_safe_mode
-    ```
-    
-4. Lastly, we can issue the certificates by running 
+1. Copy the `conf_template.ini` file and rename as `conf.ini`.
+
+1. Make sure you have enough ETH in your issuing address.
+
+   a. Go to https://myetherwallet.com and create a public address and private key. Make sure to store the private key in a secure location.
+   b. Go to https://faucet.bloxberg.org and enter your created public address to receive bergs.
+
+1. Configure the unsigned_certificates_dir variable in the conf.ini file to the unsigned certificates directory. Alternatively, you can also specify the directory created from the [cert-tools](https://github.com/bloxberg-org/cert-tools) repository to avoid copying the unsigned certificates.
+
+   ```
+   # To use a sample unsigned certificate as follows:
+   cp /cert-issuer/examples/data-testnet/unsigned_certificates/3bc1a96a-3501-46ed-8f75-49612bbac257.json /cert-issuer/data/unsigned_certificates/
+   ```
+
+1. Further configure the settings for cert-issuer in `conf.ini`. The private key file [should be](https://eth-account.readthedocs.io/en/latest/eth_account.html#eth_account.account.Account.from_key) a raw private key: a hex str, bytes or int
+
+1. Copy the folder `cert_issuer/blockchain_handlers/ethereum_sc/data` to `<virtualenv-name>/lib/<python-version>/site-packages/cert_issuer-3.0.0a3-py3.8.egg/cert_issuer/blockchain_handlers/ethereum_sc`.
```

Comment:
Obviously, this is not how to resolve this. Maybe can you explicitly tell pip to copy abi json files there?